.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```diff
@@ -60,6 +60,9 @@ body:
       description: What version of iced are you using?
       options:
         - master
+        - 0.4.2
+        - 0.4.1
+        - 0.4.0
         - 0.3.0
```

Comment:
We should just drop support for `0.3`, given it's extremely outdated.

I think we can simply list `master` and `0.4`, and ask users here to make sure they are using the latest patch version.

Reply:
Sounds good to me. Just updated the PR.